Dockerfile

```diff
@@ -13,7 +13,7 @@ ENV AIRFLOW_DAG ${AIRFLOW_HOME}/dags
 RUN mkdir -p ${AIRFLOW_DAG}
 
 # Install gosu
-ENV GOSU_VERSION 1.7
```

Comment:
We might want to change this to an `ARG` instead because there's no need for it to remain in the environment

```dockerfile
ARG GOSU_VERSION=1.10
```